src/storage-client/src/client.proto

```diff
@@ -128,8 +128,9 @@ message ProtoStorageResponse {
     repeated ProtoStatusUpdate updates = 1;
   }
 
-  message ProtoDroppedIds {
-    repeated mz_repr.global_id.ProtoGlobalId ids = 1;
+  message ProtoDroppedId {
+    reserved 1;
+    mz_repr.global_id.ProtoGlobalId id = 2;
```

Comment:
We're disabling backcompat checking for the cluster protocol types, so no need to reserve IDs here.

Reply:
Oh cool. I'll remove it then